pkg/controllers/nodepool/counter/controller.go

```diff
@@ -47,6 +48,14 @@ type Controller struct {
 	cluster    *state.Cluster
 }
 
+var WellKnownResource = sets.New(
```

Comment:
I wonder if this should just be a static resourceList that we call something like `baseResources` that we merge into the actual resource set. That might make testing easier as well